ta/pkcs11/src/processing.c

```diff
@@ -276,65 +283,3 @@ enum pkcs11_rc entry_processing_step(struct pkcs11_client *client,
 
 	return rc;
 }
-
-/*
- * entry_verify_oneshot - Run a single part verification processing
- *
- * @client = client reference
- * @ptype = Invocation parameter types
- * @params = Invocation parameters reference
- * @function - encrypt, decrypt, sign, verify, digest, ...
- * @step - update, oneshot, final
- */
-enum pkcs11_rc entry_verify_oneshot(struct pkcs11_client *client,
-				    uint32_t ptypes, TEE_Param *params,
-				    enum processing_func function,
-				    enum processing_step step)
-
-{
-	TEE_Param *ctrl = params;
-	enum pkcs11_rc rc = PKCS11_CKR_OK;
-	struct serialargs ctrlargs = { };
-	struct pkcs11_session *session = NULL;
-	enum pkcs11_mechanism_id mecha_type = PKCS11_CKM_UNDEFINED_ID;
-
-	assert(function == PKCS11_FUNCTION_VERIFY);
-	if (!client ||
-	    TEE_PARAM_TYPE_GET(ptypes, 0) != TEE_PARAM_TYPE_MEMREF_INOUT)
-		return PKCS11_CKR_ARGUMENTS_BAD;
-
-	serialargs_init(&ctrlargs, ctrl->memref.buffer, ctrl->memref.size);
-
-	rc = serialargs_get_session_from_handle(&ctrlargs, client, &session);
-	if (rc)
-		return rc;
-
-	if (serialargs_remaining_bytes(&ctrlargs))
-		return PKCS11_CKR_ARGUMENTS_BAD;
-
-	rc = get_active_session(session, function);
-	if (rc)
-		return rc;
-
-	mecha_type = session->processing->mecha_type;
-	rc = check_mechanism_against_processing(session, mecha_type,
-						function, step);
-	if (rc)
-		goto out;
-
-	if (processing_is_tee_symm(mecha_type))
-		rc = step_symm_operation(session, function, step,
-					 ptypes, params);
-	else
-		rc = PKCS11_CKR_MECHANISM_INVALID;
-
-	DMSG("PKCS11 session %"PRIu32": verify %s %s: %s", session->handle,
-	     id2str_proc(mecha_type), id2str_function(function),
-	     id2str_rc(rc));
-
-out:
-	if (rc != PKCS11_CKR_BUFFER_TOO_SMALL)
-		release_active_processing(session);
-
-	return rc;
-}
```

Comment:
If this function is not needed, it should be removed in a separate commit.

Reply:
Ok..I have created a separate patch for this.